### PR TITLE
correctly expose stats.service.interval in sys.cluster

### PR DESCRIPTION
**Pull request by m-kharbat**
should I backport to 1.0 ?

### CODE REVIEW

File: sql/src/main/java/io/crate/metadata/settings/CrateSettings.java [hunk 1]
```diff
@@ -137,11 +137,35 @@ public Setting parent() {
...
+            return STATS;
+        }
+
+        @Override
+        public boolean isRuntime() {
+            return true;
+        }
+    };
+
     public static final TimeSetting STATS_SERVICE_REFRESH_INTERVAL = new TimeSetting() {
```
Comment (chaudum) [inline sql/src/main/java/io/crate/metadata/settings/CrateSettings.java]: naming: `STATS_SERVICE_INTERVAL`

**Comment by chaudum** [comment]
@m-kharbat yes, should also be backported to 1.0 (stable) and 1.1 (testing)

**Comment by chaudum** [comment]
also added @seut for review because he has to rebase the `five` branch with these changes

**Comment by chaudum** [comment]
@m-kharbat please check test failures

**Comment by seut** [comment]
@m-kharbat pls fix itest failures (setting name must be adjusted there as well)

**Pull request merged**